packages/ui/src/helpers/authenticator/getRoute.ts

```diff
@@ -39,18 +41,20 @@ export const getRoute = (
     case actorState?.matches('confirmVerifyUserAttribute'):
       return 'confirmVerifyUser';
     case state.matches('getCurrentUser'):
-    case actorState.matches('fetchUserAttributes'):
+    case actorState?.matches('fetchUserAttributes'):
       /**
        * This route is needed for autoSignIn to capture both the
        * autoSignIn.pending and the resolved states when the
        * signIn actor is running.
        */
       return 'transition';
+    case state?.matches('signInActor.runActor'):
+      return 'signIn';
+    case state?.matches('signUpActor.runActor'):
+      return 'signUp';
```

Comment:
Putting these at the end so they don't take precedence over more relevant cases (e.g. `confirmSignUp` also has `signUpActor.runActor` state`

Reply:
Rather than use `signUpActor.runActor` as the case here can we look at the value of `actorState`? The fact that other steps (such as the noted `confirmSignUp`) have a state value of `signUpActor.runActor` makes this solution fragile IMO

Reply:
Yeah so all of the other cases I added are values of `actorState` where state is either `signUpActor.runActor` or `signInActor.runActor`. 

I included these because some customers who saw this error didn't give steps for reproduction and our current error message doesn't show `actorState` so I can't tell if the other cases are exhaustive. I did update the error message so maybe we can take these out and if anyone still sees the warning we can add a more specific case?